core-chart/values.yaml

```diff
@@ -5,11 +5,12 @@
 
 # Container/chart image versions. These values are set during the chart release process,
 # please do not change them unless you know what you are doing.
-HELM_VERSION: "3.15.0"
-KUBECTL_VERSION: "1.30.1"
+HELM_VERSION: "3.16.1"
```

Comment:
+1
(https://helm.sh/docs/topics/version_skew/)

Reply:
Perhaps, I but this has a limited usage and as long as it works, it should be ok to upgrade.

Reply:
"+1" means approval. I like that you updated the Helm version to better match our usage.